### PR TITLE
Restock Debugging Updates + Container Improvements

### PR DESCRIPTION
**Pull request by wstampley**
Making restock's new 'container' property work more like the way it intuits (i.e. it counts and stores into the same container)

@KatoakDR - Thoughts?

### CODE REVIEW

File: restock.lic [hunk 1]
```diff
@@ -36,49 +53,52 @@ class Restock
...
           bput("get #{item['name']} from counter", "You get a")
         end
-        reget(2, 'runestone') ? DRCI.put_away_item?(item['name'], @runestone_storage) : stow_hands
+
+        if item['container']
+          DRCI.put_away_item?(item['name'], item['container'])
+        elsif item['name'] =~ /runestone/i
+          DRCI.put_away_item?(item['name'], @runestone_storage)
+        else
+          stow_hands
```
Comment (KatoakDR) [inline restock.lic:85]:
```suggestion
          DRCI.stow_hands
```

**Review by KatoakDR — COMMENTED** [review]
👍  Thanks for the clean up. Code looks good to me, had one comment on a place to add `DRCI` prefix. I'll do some testing Saturday but I don't foresee any issues.

**Comment by rpherbig** [comment]
Code changes look fine to me ✔️ 

Let me know how the testing goes

**Comment by KatoakDR** [comment]
Seems to be working, here's my test results:

```yaml
restock:
  arrow:
    quantity: 5
    container: saddlebag
  bolt:
    quantity: 5
    container: thigh bag
    stackable: false
  rock:
    quantity: 5
```

```
[restock]>count my first boar-tusk arrows

You count some boar-tusk arrows in the saddlebag and see there are five left.
>
[restock]>count my second boar-tusk arrows

I could not find what you were referring to.
>
[restock]>rummage /C bolts IN MY thigh bag

You rummage through an abyssal black thigh bag looking for something similar to "bolts" and see some crossbow bolts.
>
[restock]>count my first small rocks

I could not find what you were referring to.

...

[restock]>buy crossbow bolts

You decide to purchase the bolts, and pay the sales clerk 112 Kronars.
The sales clerk hands you your crossbow bolts.

>
[restock]>put my crossbow bolts in my thigh bag

You put your bolts in your thigh bag.
>
[restock]>buy small rocks

You decide to purchase the rocks, and pay the sales clerk 12 Kronars.
The sales clerk hands you your small rocks.

>
[restock]>stow right

You put your rocks in your patched gearbag.
```

**Pull request merged**